### PR TITLE
feat: fix empty incident changelog

### PR DESCRIPTION
**Pull request by adityathebe**
Resolves: https://github.com/flanksource/flanksource-ui/issues/1257
Resolves: https://github.com/flanksource/flanksource-ui/issues/1251

### CODE REVIEW

**Comment by adityathebe** [comment]
image

File: src/components/Changelog/IncidentHistoryItemTypeContent.tsx [hunk 1]
```diff
@@ -64,8 +64,26 @@ const IncidentHistoryTypeToComponentMap = new Map<
...
+      );
+    }
+  ],
+  [
+    "evidence.done_definition_added",
+    ({ incidentHistory }) => {
+      const evidence = incidentHistory.evidence;
+      return (
+        <>
+          added done definition{" "}
```
Comment (moshloop) [inline src/components/Changelog/IncidentHistoryItemTypeContent.tsx:82]:
```suggestion
          added definition of done{" "}
```

**Comment by moshloop** [comment]
@adityathebe still seeing empty items: https://deploy-preview-1258--flanksource-demo-stable.netlify.app/incidents/0188b8b4-c580-f527-fb29-fc5e2018417a

**Comment by adityathebe** [comment]
> @adityathebe still seeing empty items: https://deploy-preview-1258--flanksource-demo-stable.netlify.app/incidents/0188b8b4-c580-f527-fb29-fc5e2018417a

@moshloop That's the old incident history which had the wrong type (`incident_status.updated`).
Fixed it

```
UPDATE incident_histories SET type = 'incident.status_updated' WHERE type = 'incident_status.updated'
```

**Comment by moshloop** [comment]
- Can we change it from added evidence {name} check to just, added check {name}
- and "added DoD for check {name} 
- Remove the avatar for responders as they cause wrapping

**Comment by adityathebe** [comment]
image

@moshloop

**Comment by mainawycliffe** [comment]
@adityathebe can we reduce the sizes of the avatars, can be achieved by passing in the `size="sm"` attribute to the avatar

image

**Comment by moshloop** [comment]
> @adityathebe can we reduce the sizes of the avatars, can be achieved by passing in the `size="sm"` attribute to the avatar
> 
> image

I think `sm` should be the default

**Comment by adityathebe** [comment]
image

@moshloop made "sm" the default avatar size.

**Comment by moshloop** [comment]
@adityathebe I think we want it rounded by default well

**Pull request merged**